src/components/Article.jsx

```diff
@@ -1 +1,19 @@
-// export const Article = () => ();
+import React from 'react';
+
+const Article = ({ title, text, date }) => (
+  <div className="box">
+    <h2 className="title" data-cy="title">
+      {title}
+    </h2>
+    <time dateTime="2019-08-30" data-cy="date">
+      {date.toLocaleDateString('en-US', { year: 'numeric',
+        month: 'long',
+        day: 'numeric' })}
```

Comment:
```suggestion
      {date.toLocaleDateString('en-US', { 
        year: 'numeric',
        month: 'long',
        day: 'numeric', 
      })}
```
It would be great to create a separate variable for this value too